From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>
Bug: https://github.com/dhall-lang/dhall-haskell/issues/2380

Disable buggy tests
All of these fail, some of these seem to require internet access to function

diff -urN dhall-1.40.2/tests/Dhall/Test/Import.hs dhall-1.40.2-r3/tests/Dhall/Test/Import.hs
--- dhall-1.40.2/tests/Dhall/Test/Import.hs	2001-09-08 19:46:40.000000000 -0600
+++ dhall-1.40.2-r3/tests/Dhall/Test/Import.hs	2022-02-08 19:18:08.199537834 -0700
@@ -47,6 +47,11 @@
             , importDirectory </> "success/unit/asLocation/RemoteChain2A.dhall"
             , importDirectory </> "success/unit/asLocation/RemoteChain3A.dhall"
             , importDirectory </> "success/unit/asLocation/RemoteChainMissingA.dhall"
+            , importDirectory </> "success/unit/asLocation/RemoteChainEnvA.dhall"
+            , importDirectory </> "success/originHeadersA.dhall"
+            , importDirectory </> "success/originHeadersImportA.dhall"
+            , importDirectory </> "success/originHeadersImportFromEnvA.dhall"
+            , importDirectory </> "success/originHeadersOverrideA.dhall"
             ]
 
     successTests <- Test.Util.discover (Turtle.chars <* "A.dhall") successTest (do
